grp/classic.gd

```diff
@@ -1413,8 +1413,8 @@ BindGlobal( "DECLARE_PROJECTIVE_SEMILINEAR_GROUPS_OPERATION",
       d:= Length( facts );
       p:= facts[1];
 
-      if d = 1 then
-        return lin;
+      if n = 1 then
+        return CyclicGroup( filt, d );
```

Comment:
Yes, the case n = 1 needs a special treatment because we need a faithful action of the field automorphisms.
I would still make a distinction for d = 1, in the sense that one does not need an additional generator in this case.

Reply:
done